8785 - Minor copy change on R&D DS&M page

anjenkin opened the pull request:
https://federal-spending-transparency.atlassian.net/browse/DA-8785

brianpetway commented:
Very tricky. The image in the comment above looks like it wants 'Office of Management and Budget (OMB)' to be a hyperlink, but the latest version of the MAC I have does not show that hyperlink.
Did you confirm that with Grace?

anjenkin commented:
@brianpetway Grace confirmed that the 'Office of Management and Budget (OMB)'  text doesn't have to be a link.  
Also, I removed the notes from DS&M per the MAC
And addressed https://federal-spending-transparency.atlassian.net/browse/DA-8858 in this PR
